packages/webpack-plugin/src/loader.ts

```diff
@@ -28,6 +28,8 @@ export default function StylableWebpackLoader(this: StylableLoaderContext, sourc
 
     const varType = this.target === 'oldie' ? 'var' : 'const';
 
+    this.onLoaderFinished();
```

Comment:
remove and move `flagStylableModule` to this location and use that as the indication